### ISSUE TITLE
Cannot determine type of resource to create from json input data: no member resourceType was found

### ISSUE THREAD

**Issue by shyam-dhanotiya** [ewoutkramer/fhir-net-api#629]
Hi,

I am new to fhir-net-api. I am getting error while calling get api 

"Cannot determine type of resource to create from json input data: no member resourceType was found"

patient example:
            var pat = new Patient();

            var id = new Identifier();
            id.System = "http://hl7.org/fhir/sid/us-ssn";
            id.Value = "000-12-3456";
            pat.Identifier.Add(id);

                   

            var name = new HumanName().WithGiven("Christopher").WithGiven("millars").AndFamily("mil");
            name.Prefix = new string[] { "Mr." };
            name.Use = HumanName.NameUse.Official;

            var nickname = new HumanName();
            nickname.Use = HumanName.NameUse.Nickname;
            nickname.GivenElement.Add(new FhirString("Chris"));

            pat.Gender = AdministrativeGender.Male;

![image](https://user-images.githubusercontent.com/20366757/42128586-8ab114ca-7c74-11e8-8303-e99988d68fd1.png)

Any help would be appreciated !

**Comment by tstolker** [comment]
Please open the same URL in the bowser and see what is being returned.

**Comment by ewoutkramer** [comment]
> Please open the same URL in the bowser and see what is being returned.

Yes, indeed, I think you will see the returned json data misses the "_resourceType" member as a child in the root object.

**Comment by shyam-dhanotiya** [comment]
Sorry for delayed response.

Could you please send a working example of custom .Net FHIR server with FHIR client Read/Write method.

**Comment by mbaltus** [comment]
You could use the http://vonk.fire.ly test server to try out your FHIR client.
If you need more help, please respond to the email I sent you from our support address.
